apps/web/src/components/Settings/Dispatcher/ToggleDispatcher.tsx

```diff
@@ -85,6 +88,31 @@ const ToggleDispatcher: FC<Props> = ({ buttonSize = 'md' }) => {
     } catch {}
   };
 
+  const handleIDKitProof = async (result: ISuccessResult) => {
+    const response = await fetch('https://world-id-lens-bridge.vercel.app/api/v1/submit', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        ...result,
+        is_production: IS_MAINNET,
+        action_id: IDKIT_ACTION_ID,
+        signal: currentProfile?.ownedBy
+      })
+    });
+
+    if (response.ok) {
+      return;
+    }
+
+    if (response.status === 400 && (await response.json()).code === 'already_verified') {
+      throw new Error(
+        'You have already verified this phone number with Lens. You can only verify one wallet with one phone number.'
+      );
+    }
+
+    throw new Error('Something went wrong. Please try again.');
```

Comment:
I think better can use `toast.error` instead of throwing an error at the app level because we don't want to log these kinds of errors in our datadog aka these are for users not for system.

Reply:
Ahh cool then 👍🏼